pkg/loader/openapi.go

```diff
@@ -136,10 +147,6 @@ func getOpenAPITools(t *openapi3.T) ([]types.Tool, error) {
 					tool.Parameters.Arguments.Properties["requestBodyContent"] = &openapi3.SchemaRef{Value: arg}
 					break
 				}
-
-				if bodyMIME == "" {
-					return nil, fmt.Errorf("no supported MIME types found for request body in operation %s", operation.OperationID)
-				}
```

Comment:
Rather than failing on the whole document if we encounter a body with no supported MIME types, we just skip it and move on.